src/js/lib/monster.js

```diff
@@ -157,6 +157,16 @@ define(function(require) {
 						eventRouting: true
 					}
 				}
+			},
+			callRecording: {
+				28: {
+					configuration: {
+						manage: true,
```

Comment:
Is this value (`callRecording.configuration.manage`) going to be read in any app? I don't see it read at [MCAR-19](https://github.com/2600hz/monster-ui-call-recording/pull/30/files)

Reply:
Oh no and I added that as Devices section falls under Configuration 🤔 

My thinking was something along the lines of `callRecording: {` similar to `smartpbx: {` with `28: {` under both of them

& `configuration: {` section under `callRecording` with  option to `manage` as we will be working on that, followed by `devices: { manage:` as the falls under Configuration

Reply:
Oh, OK, got it 👍